src/tablist.js

```diff
@@ -18,6 +18,10 @@ SideTabList.prototype = {
     if (this.alwaysShrink) {
       this.maybeShrinkTabs();
     }
+
+    this.useRegExpFilter = (await browser.storage.local.get({
```

Comment:
I see less than 10ms. Do you mean an helper function for loadPrefs? I guess it can be further reconstructed, but there is no good idea.